src/node-lib/curriculum-api-2023/queries/lessonOverview/lessonOverview.query.test.ts

```diff
@@ -182,41 +182,9 @@ describe("lessonOverview()", () => {
       ]);
     });
 
-    it("should return an array with empty copyrightInfo if not provided", () => {
```

Comment:
how come these are deleted?

Reply:
The format/schema of the data doesn't fit this test so keeps throwing type errors, copyright content should either be Null or and array with a copyright_info object, when the data is parsed if the data is incorrect it should be caught there